### ISSUE TITLE
quickgui start error

### ISSUE THREAD

**Issue by drford17** [quickemu-project/quickgui#68]
Hi guys,

I am running Fedora35 and tried to launch the quickgui binary with the following error:

` !  ~/.a/quickgui-1.2.3  ./quickgui    360ms  Fri 25 Feb 2022 12:51:24 AM CET
[ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: ProcessException: No such file or directory
  Command: quickget list_csv
#0      _ProcessImpl._start (dart:io-patch/process_patch.dart:401)
#1      Process.start (dart:io-patch/process_patch.dart:38)
#2      _runNonInteractiveProcess (dart:io-patch/process_patch.dart:578)
#3      Process.run (dart:io-patch/process_patch.dart:49)
#4      loadOperatingSystems (package:quickgui/main.dart:16)
#5      main (package:quickgui/main.dart:61)
#6      main (file:///opt/flutter-projects/quickgui/.dart_tool/flutter_build/generated_main.dart)
#7      _runMainZoned.<anonymous closure>.<anonymous closure> (dart:ui/hooks.dart:127)
#8      _rootRun (dart:async/zone.dart:1428)
#9      _CustomZone.run (dart:async/zone.dart:1328)
#10     _runZoned (dart:async/zone.dart:1863)
#11     runZonedGuarded (dart:async/zone.dart:1851)
#12     _runMainZoned.<anonymous closure> (dart:ui/hooks.dart:125)
#13     _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:297)
#14     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:192)
`

Can someone help?

Thanks a lot

**Comment by ymauray** [comment]
Hi, thanks for the report. The binary has been re-built, and version 1.2.4-1 has been pushed to the PPA. Can you update and let me know if this issue is resolved ? Thank !

**Comment by floviolleau** [comment]
Hi,

I have the same issue on Ubuntu 21.10.

```
flo@toto $ quickgui -h
[ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: ProcessException: No such file or directory
  Command: quickget list_csv
#0      _ProcessImpl._start (dart:io-patch/process_patch.dart:401)
#1      Process.start (dart:io-patch/process_patch.dart:38)
#2      _runNonInteractiveProcess (dart:io-patch/process_patch.dart:578)
#3      Process.run (dart:io-patch/process_patch.dart:49)
#4      loadOperatingSystems (package:quickgui/main.dart:16)
#5      main (package:quickgui/main.dart:61)
#6      main (file:///home/yannick/quickgui/.dart_tool/flutter_build/generated_main.dart)
#7      _runMainZoned.<anonymous closure>.<anonymous closure> (dart:ui/hooks.dart:127)
#8      _rootRun (dart:async/zone.dart:1428)
#9      _CustomZone.run (dart:async/zone.dart:1328)
#10     _runZoned (dart:async/zone.dart:1863)
#11     runZonedGuarded (dart:async/zone.dart:1851)
#12     _runMainZoned.<anonymous closure> (dart:ui/hooks.dart:125)
#13     _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:297)
#14     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:192)

flo@toto $ sudo apt-cache policy quickgui 
quickgui:
  Installed: 1.2.4-1~impish1.0
  Candidate: 1.2.4-1~impish1.0
  Version table:
 *** 1.2.4-1~impish1.0 500
        500 http://ppa.launchpad.net/yannick-mauray/quickgui/ubuntu impish/main amd64 Packages
        100 /var/lib/dpkg/statu
```

**Comment by ymauray** [comment]
Hi. 

Do you have quickget installed, and in the path ? 

What do you get when you type `quickget list_csv` in the terminal ?

By the way, there is no `-h` parameter to `quickgui` :p

**Comment by floviolleau** [comment]
Hi,

I'm on an Ubuntu impish (21.10).
I installed it via packages (apt install quickget) so it is automatically in the path.

In the stacktrace, I can see an hardcoded path to `file:///home/yannick/quickgui/.dart_tool/flutter_build/generated_main.dart` which is not me definitely.

quickget is not available at all

Regarding `-h`, okey :). If I don't put it, same issue.

Thanks

**Comment by seltzered** [comment]
Had the same issue (on Ubuntu 21.10) as @floviolleau but installing quickemu (following directions in https://github.com/quickemu-project/quickemu#install-quickemu ) resolved it. @floviolleau did you install quickemu in addition to quickget?

I'm not too familiar with apt packaging, guessing it's rather complicated to point to quickemu apt repo/package as a dependency since it's on another repository path?

In any case, thanks for this tool!

stacktrace just as an fyi:
```
Gtk-Message: 17:40:44.092: Failed to load module "appmenu-gtk-module"
[ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: ProcessException: No such file or directory
  Command: quickget list_csv
#0      _ProcessImpl._start (dart:io-patch/process_patch.dart:401)
#1      Process.start (dart:io-patch/process_patch.dart:38)
#2      _runNonInteractiveProcess (dart:io-patch/process_patch.dart:578)
#3      Process.run (dart:io-patch/process_patch.dart:49)
#4      loadOperatingSystems (package:quickgui/main.dart:16)
#5      main (package:quickgui/main.dart:61)
#6      main (file:///home/yannick/quickgui/.dart_tool/flutter_build/generated_main.dart)
#7      _runMainZoned.<anonymous closure>.<anonymous closure> (dart:ui/hooks.dart:127)
#8      _rootRun (dart:async/zone.dart:1428)
#9      _CustomZone.run (dart:async/zone.dart:1328)
#10     _runZoned (dart:async/zone.dart:1863)
#11     runZonedGuarded (dart:async/zone.dart:1851)
#12     _runMainZoned.<anonymous closure> (dart:ui/hooks.dart:125)
#13     _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:297)
#14     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:192)
```

**Comment by floviolleau** [comment]
@seltzered I installed only quickgui

**Comment by flexiondotorg** [comment]
This issue seems to be caused when Quickgui tried to start a VM but an executable `quickemu` is not in the PATH.

**Comment by floviolleau** [comment]
> This issue seems to be caused when Quickgui tried to start a VM but an executable `quickemu` is not in the PATH.

Indeed. When installing quickemu properly, it works. Issue can be closed

Thanks!

**Comment by flexiondotorg** [comment]
I believe this is closed via #74
